Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -53,6 +53,7 @@
     private TagEditText mTagEditText;    /* Chips view editor for tags */
 
     private int         mBackgroundColor;
+    private boolean     mNoteChanged;
```

Comment:
I will change the alert dialog like that.

The need for `isContentChanged()` is [here](https://github.com/vishesh/sealnote/blob/master/Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java#L462)